src/dev/run_find_plugins_without_ts_refs.ts

```diff
@@ -88,7 +89,7 @@ function isMigratedToTsProjectRefs(dir: string): boolean {
   try {
     const path = Path.join(dir, 'tsconfig.json');
     const content = Fs.readFileSync(path, { encoding: 'utf8' });
-    return get(JSON.parse(content), 'compilerOptions.composite', false);
+    return get(JSON5.parse(content), 'compilerOptions.composite', false);
```

Comment:
What was this change needed for?

Reply:
Oops, Mikhail's got a [PR open for that fix](https://github.com/elastic/kibana/pull/87658/files).

Reply:
There was a bug in the script, I added the changes and was apparently a little too eager to submit this PR 😆

Reply:
Ah, ok - that makes sense.

Reply:
@tylersmalley I removed that.